Components/Card/card.css

```diff
@@ -0,0 +1,189 @@
+/* Basic Card */
+.basic-card {
+  max-width: 360px;
+  background-color: white;
+  box-shadow: 0px 0px 3px 1px rgba(0, 0, 0, 0.3);
+  font-family: "Poppins", sans-serif;
+}
+
+.basic-card-primary {
+  width: 92%;
+  margin: 0 auto;
+}
+
+.basic-card-primary-h2 {
+  color: black;
```

Comment:
You can use variables for color instead of directly using color names or hex codes so that you can reuse the color whenever you need and if you need to change it, you will have to change at one place only.